Source/CesiumRuntime/Private/CesiumTextureResource.h

```diff
@@ -110,7 +111,7 @@ class FCesiumUseExistingTextureResource : public FCesiumTextureResourceBase {
 class FCesiumCreateNewTextureResource : public FCesiumTextureResourceBase {
 public:
   FCesiumCreateNewTextureResource(
-      CesiumGltf::ImageCesium&& image,
+      CesiumGltf::ImageCesium& image,
```

Comment:
The passed image is still being moved away, so the `&&` is more accurate. Why the change?